client/lib/signup/step-actions/index.js

```diff
@@ -170,6 +172,10 @@ function getNewSiteParams( {
 			site_creation_flow: flowToCheck,
 			timezone_string: guessTimezone(),
 			wpcom_public_coming_soon: 1,
+			difm_lite_site_options: {
+				difm_lite_site_category: selectedDIFMCategory || undefined,
+				difm_lite_typeform_response_id: typeformResponseId || undefined,
+			},
```

Comment:
I have a gut feeling that it'd be better to make `difm_lite_site_options` a child of the `options` param. But no real justification for why. It just seems like that's where we put the grab-bag of options we want to set on the newly created site.

For the other flows this is going to send `difm_lite_site_options: {}` as part of the request body. It seems tidier to leave it out of the payload when it's not the DIFM flow. I guess by only sending the options when it's non-empty? Not a big deal though.

Reply:
I am actually using `difm_lite_site_options` as a child prop of options :)
![image](https://user-images.githubusercontent.com/3422709/145048905-be08b046-c152-40ae-b6be-098564c5a073.png)

Reply:
Curse you GitHub diff view :shake-fist: ! 😆